### PR TITLE
add moni

### PR DESCRIPTION
**Pull request by AndreaGuarracino**

### CODE REVIEW

**Comment by AndreaGuarracino** [comment]
Thank you @martin-g! At the end, I was able to win \o/ Could you please review and merge this?

**Comment by AndreaGuarracino** [comment]
@bioconda-bot please add label

File: recipes/moni/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+c_compiler_version: # [unix]
```
Comment (martin-g) [inline recipes/moni/conda_build_config.yaml:1]:
```suggestion
c_compiler_version:
```

File: recipes/moni/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  run_exports:
+      - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - git
+    - zlib
```
Comment (martin-g) [inline recipes/moni/meta.yaml:24]:
```suggestion
```

**Review by martin-g — APPROVED** [review]

**Pull request merged**